docs/ingestion/native-batch.md

```diff
@@ -192,7 +192,7 @@ that range if there's some stray data with unexpected timestamps.
 |--------|-----------|-------|---------|
 |type|The task type, this should always be `index_parallel`.|none|yes|
 |inputFormat|[`inputFormat`](./data-formats.md#input-format) to specify how to parse input data.|none|yes|
-|appendToExisting|Creates segments as additional shards of the latest version, effectively appending to the segment set instead of replacing it. The current limitation is that you can append to any datasources regardless of their original partitioning scheme, but the appended segments should be partitioned using the `dynamic` partitionsSpec.|false|no|
+|appendToExisting|Creates segments as additional shards of the latest version, effectively appending to the segment set instead of replacing it. This means that you can append new segments to any datasource regardless of its original partitioning scheme. However, you must use the `dynamic` partitioning type for the appended segments .|false|no|
```

Comment:
The doc should address what happens if you disregard this advice and set appendToExisting: true but don't use dynamic partitioning. (Do you get an error? Does something weird happen?)

Reply:
I am still a noob, but I tried setting the partitionspec to `hashed` and appendToExisting to `true`. The task succeeded, but when I check the payload it looks like appendToExisting was set to `false`
```
    "ioConfig": {
      "type": "index_parallel",
      "inputSource": {
        "type": "http",
        "uris": [
          "https://static.imply.io/data/wikipedia.json.gz"
        ],
        "httpAuthenticationUsername": null,
        "httpAuthenticationPassword": null
      },
      "inputFormat": {
        "type": "json",
        "flattenSpec": {
          "useFieldDiscovery": true,
          "fields": []
        },
        "featureSpec": {}
      },
      "appendToExisting": false
```
I'll need to try again maybe and follow up with someone with deeper insight to verify how it is supposed to work in this case.

Reply:
@jihoonson PTAL. Clarified that the task will fail if you use another partitioning type and why. Included the error text in case someone needs to search on that. Thanks.